F/faust/build_tarballs.jl

```diff
@@ -0,0 +1,188 @@
+    FileProduct(joinpath("bin", "faust2pdf"), :faust2pdf),
+    FileProduct(joinpath("bin", "faust2plot"), :faust2plot),
+    FileProduct(joinpath("bin", "faust2png"), :faust2png),
+    FileProduct(joinpath("bin", "faust2portaudiorust"), :faust2portaudiorust),
+    FileProduct(joinpath("bin", "faust2puredata"), :faust2puredata),
+    FileProduct(joinpath("bin", "faust2pure"), :faust2pure),
+    FileProduct(joinpath("bin", "faust2raqt"), :faust2raqt),
+    FileProduct(joinpath("bin", "faust2ros"), :faust2ros),
+    FileProduct(joinpath("bin", "faust2rosgtk"), :faust2rosgtk),
+    FileProduct(joinpath("bin", "faust2rpialsaconsole"), :faust2rpialsaconsole),
+    FileProduct(joinpath("bin", "faust2rpinetjackconsole"), :faust2rpinetjackconsole),
+    FileProduct(joinpath("bin", "faust2sam"), :faust2sam),
+    FileProduct(joinpath("bin", "faust2sc"), :faust2sc),
+    FileProduct(joinpath("bin", "faust2sig"), :faust2sig),
+    FileProduct(joinpath("bin", "faust2sigviewer"), :faust2sigviewer),
+    FileProduct(joinpath("bin", "faust2smartkeyb"), :faust2smartkeyb),
+    FileProduct(joinpath("bin", "faust2sndfile"), :faust2sndfile),
+    FileProduct(joinpath("bin", "faust2soul"), :faust2soul),
+    FileProduct(joinpath("bin", "faust2supercollider"), :faust2supercollider),
+    FileProduct(joinpath("bin", "faust2svg"), :faust2svg),
+    FileProduct(joinpath("bin", "faust2svgviewer"), :faust2svgviewer),
+    FileProduct(joinpath("bin", "faust2teensy"), :faust2teensy),
+    FileProduct(joinpath("bin", "faust2unity"), :faust2unity),
+    FileProduct(joinpath("bin", "faust2vcvrack"), :faust2vcvrack),
+    FileProduct(joinpath("bin", "faust2vst"), :faust2vst),
+    FileProduct(joinpath("bin", "faust2vsti"), :faust2vsti),
+    FileProduct(joinpath("bin", "faust2w32max6"), :faust2w32max6),
+    FileProduct(joinpath("bin", "faust2w32msp"), :faust2w32msp),
+    FileProduct(joinpath("bin", "faust2w32puredata"), :faust2w32puredata),
+    FileProduct(joinpath("bin", "faust2w32vst"), :faust2w32vst),
+    FileProduct(joinpath("bin", "faust2w64max6"), :faust2w64max6),
+    FileProduct(joinpath("bin", "faust2w64vst"), :faust2w64vst),
+    FileProduct(joinpath("bin", "faust2wasm"), :faust2wasm),
+    FileProduct(joinpath("bin", "faust2webaudiowasm"), :faust2webaudiowasm),
+    FileProduct(joinpath("bin", "faust2webaudiowast"), :faust2webaudiowast),
+    FileProduct(joinpath("bin", "faust2winunity"), :faust2winunity), 
+
+    FileProduct(joinpath("bin","faustoptflags"), :faustoptflags),
+    FileProduct(joinpath("bin","usage.sh"), :usage_sh),
+
+    FileProduct(joinpath("share", "faust", "stdfaust.lib"), :stdfaust_lib),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("libLLVM_jll", v"11.0.1"),
+    Dependency("libmicrohttpd_jll"),
+    Dependency("libsndfile_jll"),
+    BuildDependency("Ncurses_jll"),
+    BuildDependency("XML2_jll"),
+    BuildDependency("Zlib_jll"),
```

Comment:
Just to confirm, these libraries aren't needed at runtime?

Reply:
> Also to clarify, libLLVM is a runtime dependency of the Faust compiler. A HostBuildDependency wouldn't work in that case, correct?

You can have it both as `HostBuildDependency` and `Dependency`, and use `llvm-config` from `${host_prefix}/tool`

Reply:
Seems like the build isn't happy with just `libLLVM` as a `Dependency`, instead of `LLVM`. Changing it back to `LLVM` and adding a `HostBuildDependency` of the same gives the following error:
```
[11:56:02] /opt/aarch64-linux-gnu/bin/../lib/gcc/aarch64-linux-gnu/8.1.0/../../../../aarch64-linux-gnu/bin/ld: skipping incompatible /opt/x86_64-linux-musl/lib/libLLVM-11.so when searching for -lLLVM-11
[11:56:02] /opt/aarch64-linux-gnu/bin/../lib/gcc/aarch64-linux-gnu/8.1.0/../../../../aarch64-linux-gnu/bin/ld: cannot find -lLLVM-11
[11:56:02] collect2: error: ld returned 1 exit status
```

Maybe it picked up the wrong `llvm-config` in the path? Regardless, it seems better to just rely on `LLVMConfig.cmake`, which should have all the right values for the target platform.

Reply:
Ok, ok, thanks!  Great work overall!